.github/pull_request_template.md

```diff
@@ -0,0 +1,3 @@
+## Stack
```

Comment:
Yep, just so that we can have a header for the visualisation. I do plan to eventually include checklists/additional instructions in the PR template as well to aid contributors.